src/main/java/com/jagrosh/jmusicbot/commands/general/SettingsCmd.java

```diff
@@ -59,7 +59,7 @@ protected void execute(CommandEvent event)
                         + "\nVoice Channel: " + (vchan == null ? "Any" : "**" + vchan.getName() + "**")
                         + "\nDJ Role: " + (role == null ? "None" : "**" + role.getName() + "**")
                         + "\nCustom Prefix: " + (s.getPrefix() == null ? "None" : "`" + s.getPrefix() + "`")
-                        + "\nRepeat Mode: **" + (s.getRepeatMode() ? "On" : "Off") + "**"
+                        + "\nRepeat Mode: **" + s.getRepeatMode() + "**"
```

Comment:
Shall we make it bold when the repeat mode is not off instead?

Reply:
Yeah